### PR TITLE
Prisma pb update

### PR DESCRIPTION
**Pull request by ayman-m**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added a new classification rule for the prisma policy: "AWS Security Group allows all traffic on SSH port (22)" with ID: "617b9138-584b-4e8e-ad15-7fbabafbed1a"
- Fixed the "Account Name" mapping under the "AWS EC2 Instance Misconfiguration" incident type
- Updated the playbook to investigate the "AWS Security Group allows all traffic on SSH port (22)" alerts
- Updated the playbook to investigate the "AWS Security Group allows all traffic on SSH port (22)" alerts

## Minimum version of Cortex XSOAR
- [ x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ ] Tests
- [ x] Documentation

### CODE REVIEW

File: Packs/PrismaCloud/ReleaseNotes/2_0_10.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#### Mappers
+##### Prisma Cloud - Incoming Mapper
+- Fixed the "Account Name" mapping under the "AWS EC2 Instance Misconfiguration" incident type
+
+#### Playbooks
+##### Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration
+- Updated the playbook to investigate the "AWS Security Group allows all traffic on SSH port (22)" alerts
+
+##### Prisma Cloud Remediation - AWS EC2 Security Group Misconfiguration
+- Updated the playbook to investigate the "AWS Security Group allows all traffic on SSH port (22)" alerts
```
Comment (ShacharKidor) [inline Packs/PrismaCloud/ReleaseNotes/2_0_10.md:13]: We have a new validation for the release notes that checks that every section starts with one of the following prefixes: 

```RN_TEMPLATES = {
    # prefixes
    'Added support for',
    'Added the',
    'Fixed an issue',
    'Improved implementation',
    'Updated the Docker image to',
    'You can now',
    'Deprecated. ',
    'Deprecated the ',

    # full line
    'Documentation and metadata improvements.',
}

BANNED_TEMPLATES = {
    'maintenance and stability enhancements.',
    'stability and maintenance enhancements.',
}
```


Can you please edit your release notes in accordance with this new validation?

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @ayman-m, 
Thank you for contributing!

I asked for one little change in the release notes, please try to adjust them.
I'll assign @idovandijk for a review as well.

 I'm available for every question here or via slack :)

File: Packs/PrismaCloud/ReleaseNotes/2_0_10.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#### Classifiers
+- **Prisma Cloud - Classifier**
```
Comment (ShacharKidor) [inline Packs/PrismaCloud/ReleaseNotes/2_0_10.md:2]:
```suggestion
#### Prisma Cloud - Classifier
```

**Review by ShacharKidor — APPROVED** [review]
@ayman-m - Great job !! :)

**Comment by ShirleyDenkberg** [comment]
@ShacharKidor @idovandijk Doc review completed.

File: Packs/PrismaCloud/ReleaseNotes/2_0_10.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#### Classifiers
+- **Prisma Cloud - Classifier** - Added the new classification rule: "617b9138-584b-4e8e-ad15-7fbabafbed1a" to classify the prisma policy: "AWS Security Group allows all traffic on SSH port (22)" alerts as "AWS EC2 Instance Misconfiguration" incidents.
+#### Mappers
+##### Prisma Cloud - Incoming Mapper
+Fixed an issue with the "Account Name" mapping under the "AWS EC2 Instance Misconfiguration" incident type.
```
Comment (ShacharKidor) [inline Packs/PrismaCloud/ReleaseNotes/2_0_10.md:5]:
```suggestion
#### Classifiers
##### Prisma Cloud - Classifier
Added the new classification rule: "617b9138-584b-4e8e-ad15-7fbabafbed1a" to classify the prisma policy: "AWS Security Group allows all traffic on SSH port (22)" alerts as "AWS EC2 Instance Misconfiguration" incidents.

#### Mappers
##### Prisma Cloud - Incoming Mapper
Fixed an issue with the "Account Name" mapping under the "AWS EC2 Instance Misconfiguration" incident type.
```

**Comment by idovandijk** [comment]
Hi, unfortunately I will need a few more days to review this PR. I apologize for the delay, and thanks for waiting.

**Review by idovandijk — CHANGES_REQUESTED** [review]
Thank you @ayman-m for this contribution!
After reviewing your work, I'm happy to say that you've incorporated the changes pretty well!
You've only left me with 4 points to ask that you please address:

1. I see that the mapping for Account Name for the `AWS IAM Policy Misconfigureation` incident type remains `resource.accountname`. So: are you sure your change to `resource.account` is correct, and if so could you please also fix that one?
image

2. In `Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration`, please use the `IsIntegrationAvailable` to check if the integration is enabled - that's our best practice and it available for use since XSOAR 5.0.0
3. Same as 2 for `Prisma Cloud Remediation - AWS EC2 Security Group Misconfiguration`
4. In `Prisma Cloud Remediation - AWS EC2 Security Group Misconfiguration`, task 30 `Revoke the 22 ingress rule`, you are stopping on errors, so you would not get to task 31 unless manually proceeded by the user. Please change that to be like the rest: 
image

Also I would really like to see an incident actually running with this just to ensure that it all runs smoothly. Could we schedule a short demo?
Thanks again

**Pull request merged**